### ISSUE TITLE
User created boards like 8chan

### ISSUE THREAD

**Issue by videah** [karai17/lapis-chan#21]
8chan allows user created boards with moderation tools and custom css, making it a sort of reddit/imageboard hybrid. Having this as an option in lapis-chan would be nice.

**Comment by karai17** [comment]
I agree. The admin interface of lapis-chan is pretty garbage, but (mostly) functional. I definitely want to revamp it sooner rather than later, so perhaps adding more global settings and allow people to sign up and switch permissions to be board-based instead of global (but maintaining global too) could be a better direction for lapis-chan.

I am 100% open to completely reworking this.
